sunset/code/game/objects/structures/beds_chairs/bed.dm

```diff
@@ -0,0 +1,66 @@
+/obj/structure/bed/patient
+	name = "Patient Bed"
+	icon = 'sunset/icons/obj/objects.dmi'
+	icon_state = "medbed"
+	desc = "For when patients want to get comfortable while their medical insurance takes forever to go through."
+	anchored = TRUE
+	resistance_flags = FLAMMABLE
+	var/mutable_appearance/bedrest
+
+/obj/structure/bed/patient/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/screwdriver))
+		if(bolts)
+			to_chat(user, "<span class='warning'>You loosen up the bolts in the wheels.</span>")
+			bolts = FALSE
+			anchored = FALSE
+			return
+
+		else
+			to_chat(user, "<span class='warning'>You tighten the bolts in the wheels.</span>")
+			bolts = TRUE
+			anchored = TRUE
+			return
+	else
+		return ..()
+
+/obj/structure/bed/patient/post_buckle_mob(mob/living/M)
+	M.pixel_y = initial(M.pixel_y)
+	update_icon()
+
+/obj/structure/bed/patient/Moved()
+	. = ..()
+	if(has_gravity())
+		playsound(src, 'sound/effects/roll.ogg', 100, 1)
+
+/obj/structure/bed/patient/post_unbuckle_mob(mob/living/M)
+	M.pixel_x = M.get_standard_pixel_x_offset(M.lying)
+	M.pixel_y = M.get_standard_pixel_y_offset(M.lying)
+
+/obj/structure/bed/patient/Initialize()
+	bedrest = Getbedrest()
```

Comment:
There is no need for a proccall as it dosent have any subtype

```suggestion
	bedrest = mutable_appearance('sunset/icons/obj/objects.dmi', "medbed_overlay")
```

Reply:
Can you specify on that?